extra/dump-gitlab-env.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+import os
+import requests
+import argparse
+
+GITLAB_TOKEN = os.environ.get("GITLAB_TOKEN")
+if GITLAB_TOKEN is None:
+    raise RuntimeError("provide the GITLAB_TOKEN variable!")
+
+# not every var is interesting for local dev
+DEV_VARS = ["STRIPE_API_KEY", "AWS_SECRET_ACCESS_KEY", "AWS_ACCESS_KEY_ID"]
```

Comment:
The `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` are not testing keys: they are the ones used to publish deliverables into S3. So why do we need them for local testing? We should then have two pairs of keys as we do in other Pipelines, and this tool should only retrieve the one with lower privileges.